uint8 digital_in_pin   # Digital input pin of ur_contoroler
bool state             # Pin state. True : on,  False : off